src/Symfony/Component/DependencyInjection/composer.json

```diff
@@ -29,6 +29,7 @@
     "suggest": {
         "symfony/yaml": "",
         "symfony/config": "",
+        "symfony/expression-language": "For using expressions in service container configuration files",
```

Comment:
I would remove ``files``. This is true even when configuring the container programmatically rather than using config files